test/runtests.jl

```diff
@@ -216,6 +216,46 @@ end
 
 rm(joinpath(Documenter_root, "build"); recursive = true)
 
+# Try generating documentation with generate()
+
+pkg_generate = if VERSION >= v"0.5-"
+    Pkg.add("PkgDev")
```

Comment:
I don't see that interestingly.. the "Nothing to be done" pops out quite quickly. But we can have that check just in case.

I was actually wondering -- is there a better way of handling deps for tests?